lib/aliquot-pay.rb

```diff
@@ -155,7 +144,9 @@ def build_signed_key
     return @signed_key if @signed_key
     ensure_intermediate_key
 
-    if !@intermediate_key.private_key? && !@intermediate_key.public_key?
+    if @intermediate_key.private_key? || @intermediate_key.public_key?
+      public_key = @intermediate_key
+    else
```

Comment:
I'm sorry but I don't see the point in this change. I might be blind of course :slightly_smiling_face: 

The variable `public_key` is not used afterwards. Perhaps just roll back to the original lines?

Another thing that's more important: `eckey_to_public` is used in line 153, and the method has been removed.

Reply:
https://github.com/clearhaus/aliquot-pay/pull/12/commits/dd8a99773cb26e75263faaa711b00f4624aaf6f8

Reply:
Done :smile: but I see you have made the related changes in [`dd8a997` (#12)](https://github.com/clearhaus/aliquot-pay/pull/12/commits/dd8a99773cb26e75263faaa711b00f4624aaf6f8)

Reply:
Somehow rolling back your change made specs fail :exploding_head: :sweat_smile:

Reply:
Aah, I think you forgot a couple of exclamation marks :wink: